test/fixtures/output/kafka/link/describe.golden

```diff
@@ -0,0 +1,8 @@
++---------------------+--------------------+
+| Name                | link-1             |
+| Source Cluster      | lkc-describe-topic |
+| Destination Cluster |                    |
+| State               | AVAILABLE          |
+| Error               |                    |
+| Error Message       |                    |
```

Comment:
`omitempty`?

Reply:
I ended up making a second struct just for describe that has omitempty on these two fields.

Turns out that while omitempty is ignored in the human list output, it's used in the json and yaml list output.